src/plugins/blocksmith/components/document-block-modal/index.jsx

```diff
@@ -139,7 +136,7 @@ class DocumentBlockModal extends React.Component {
   }
 
   handleChangeAspectRatio(option) {
-    this.setState({ aspectRatio: option ? AspectRatioEnum.create(option.value) : AspectRatioEnum.create('auto') });
+    this.setState({ aspectRatio: option ? AspectRatioEnum.create(option.value) : AspectRatioEnum.AUTO });
```

Comment:
@efjacobson - now that we always have a value for `aspectRatio` is the check even needed here?